tophatmonocle/hivemind-service
# Feature/match endpoint

- Created `match` GET endpoint template

DevQA:
- Make a GET request to `http://127.0.0.1:8888/linkedin_profile_data/match` without any params
- Expect to see a response with a status==400 bad request and an error message
- Add a param `li_url` with an arbitrary url
- Expect to see a response with a status==200 bad request without any error messages
